functions/go/set-namespace/Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM golang:1.14
+FROM golang:1.13-stretch
```

Comment:
What are the binary files (set-namespace-linux) being checked in?

Reply:
That follows the old set-namespace function. It has 2 binary files checked in, one for linux and one for mac.

Reply:
There's a functions/go/set-namespace/set-namespace-linux which is being added in the new function, no? In general, we should build from source, and not check in any prebuilts in Git.

Reply:
I guess those old binary files are checked in for exec test. Since we will avoid non-containerized tests so I will remove it.

Reply:
IMO binary should not be checked in here.
And even if we want to check in some big files, GIt LFS should be used.